spyder/plugins/onlinehelp/widgets.py

```diff
@@ -4,25 +4,50 @@
 # Licensed under the terms of the MIT License
 # (see spyder/__init__.py for details)
 
-"""pydoc widget"""
+"""
+PyDoc widget.
+"""
 
 # Standard library imports
-import os.path as osp
+import os
 import pydoc
 import sys
 
 # Third party imports
-from qtpy.QtCore import Qt, QThread, QUrl, Signal
+from qtpy.QtCore import Qt, QThread, QUrl, Signal, Slot
 from qtpy.QtGui import QCursor
-from qtpy.QtWidgets import QApplication
+from qtpy.QtWebEngineWidgets import WEBENGINE
+from qtpy.QtWidgets import QApplication, QLabel, QVBoxLayout
 
 # Local imports
-from spyder.config.base import _
-from spyder.plugins.onlinehelp.pydoc_patch import (
-    _start_server, _url_handler)
-from spyder.py3compat import PY3, to_text_string
+from spyder.api.translations import get_translation
+from spyder.api.widgets import PluginMainWidget
+from spyder.plugins.onlinehelp.pydoc_patch import (CustomHTMLDoc,
+                                                   _start_server, _url_handler)
 from spyder.utils.misc import select_port
-from spyder.widgets.browser import WebBrowser
+from spyder.widgets.browser import WebView, WebViewActions
+from spyder.widgets.comboboxes import UrlComboBox
+from spyder.widgets.findreplace import FindReplace
+
+
+# Localization
+_ = get_translation('spyder')
+
+
+# --- Constants
+# ----------------------------------------------------------------------------
+PORT = 30128
+PORT = 7464

```

Comment:
There are two port definitions here. Please leave only the first one.

Reply:
Oops, will do!

Reply:
Fixed